onnxruntime/core/session/plugin_ep/ep_api.cc

```diff
@@ -230,6 +258,9 @@ static constexpr OrtEpApi ort_ep_api = {
     &OrtExecutionProviderApi::SyncStream_GetImpl,
     &OrtExecutionProviderApi::SyncStream_GetSyncId,
     &OrtExecutionProviderApi::GetSyncIdForLastWaitOnSyncStream,
+
```

Comment:
```suggestion
    // End of Version 23 - DO NOT MODIFY ABOVE

```

Reply:
This was added in a separate PR today: https://github.com/microsoft/onnxruntime/pull/26418